project_sequence/views/project_project.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0) -->
+<data>
+
+<record id="project_sequence_form_edit_project" model="ir.ui.view">
+    <field name="name">Project_sequence_project_edit</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.edit_project" />
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='partner_id']" position="after">
+            <field name="code" />
+            </xpath>
+    </field>
+    </record>
+    <record id="project_sequence_form_view_project" model="ir.ui.view">
+        <field name="name">Project_sequence_project_view</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.view_project" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_id']" position="after">
+                <field name="display_name" />
+                <field name="code" optional="hide" />
+                </xpath>
+
+        </field>
+</record>
+<record id="project_sequence_form_view_project_search" model="ir.ui.view">
+    <field name="name">Project_sequence_project_view_search</field>
+    <field name="model">project.project</field>
+    <field name="inherit_id" ref="project.view_project_project_filter" />
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='partner_id']" position="after">
+                <field name="code" />
```

Comment:
Use consistent indentation please:

```suggestion
            <field name="code" />
```

BTW, it was expected that the *default* search included the code. If the default search is using `display_name`, that'll be the case automatically (because the code's always there).